pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/con_pantheon_yaml_file.adoc

```diff
@@ -0,0 +1,45 @@
+[id='con-pantheon-yaml-file_{context}']
+
+= The Pantheon configuration file
+
```

Comment:
Perhaps an introductory statement about what is a pantheon configuration file here?

Reply:
I thought that's what the first sentence is. What do you suggest?

Reply:
Something on the lines of "A Pantheon configuration file is a yaml file containing configuration information required by Pantheon for publishing modules".

The reason for this suggestion is that when a user read the title "The configuration file", the reader would expect the following text to say what is the configuration file. The first sentence, however, is about using a Git repository to import content into pantheon. Reversing the clauses, such as "You need to create a configuration file, named `pantheon2.yml` in the root directory .." should suffice though. Although, I'm not a big fan of having instructions in a conceptual topic - but that's just my thing!

Reply:
I made the change you suggested. Please take a look at my changes:
> You need to create a configuration file, named `pantheon2.yml`, in the root directory of your repository, to enable a Git repository as a source that Pantheon can use to import content. This yaml file provides necessary information that Pantheon uses to interact with files in the repository.

I understand what you're saying about not including instructions in a conceptual file. I don't really see these as instructions, per se. It's more a statement that this is something that is required, and it requires you to take action to do it. It doesn't tell you how to do it.

Reply:
Agreed.